internal/apiserver/route_get_identity_by_did.go

```diff
@@ -25,22 +25,25 @@ import (
 	"github.com/hyperledger/firefly/pkg/core"
 )
 
-var postNewNamespace = &oapispec.Route{
-	Name:   "postNewNamespace",
-	Path:   "namespaces",
-	Method: http.MethodPost,
+var getIdentityByDID = &oapispec.Route{
+	Name:   "getIdentityByDID",
+	Path:   "identities/{did:did:.+}",
```

Comment:
Why is this `did:did:` @awrichar?

I'm assuming to query for `did:firefly:org/myorg` I'm going to query:
`/api/v1/identities/did:firefly:org/myorg` (_not_ `/api/v1/identities/did:did:firefly:org/myorg`)

Reply:
oh, got it - being dim. This is the Gorilla mux syntax confusing me - the first `did:` is saying "Treat this as a path param called `did`" 🤦

Reply:
Yea it reads a bit weird, but the regex matching `did:.+` will map into a gorilla path param named "did".

Reply:
Wonder if I could make that more obvious 😅

Reply:
Think it's good as it is.